airdrop/project/contracts/distributor-contract/src/interface.sw

```diff
@@ -113,14 +113,3 @@ abi Info {
     #[storage(read)]
     fn number_of_leaves() -> u64;
 }
-
-abi SimpleAsset {
-    /// Mints the given amount of tokens to the given Identity.
-    ///
-    /// # Arguments
-    ///
-    /// * `amount`: [u64] - The quantity of tokens to mint.
-    /// * `to`: [Identity] - The user which will receive the minted tokens.
-    #[storage(read, write)]
-    fn mint_to(amount: u64, to: Identity);
```

Comment:
Why is this removed?

Reply:
This is legacy code left over from a previous test suite. Should never have been here